app/src/main/java/ch/epfl/sweng/erpa/services/GameService.java

```diff
@@ -6,13 +6,11 @@
 
 import ch.epfl.sweng.erpa.model.Game;
 
-public interface GameService
+public interface GameService extends DataService<Game>
 {
-    //constants
     String EXTRA_GAME_KEY = "game|";
 
-    //methods
-    Optional<Game> getGame(String gid);
-    Set<Game> getAll();
+    Optional<Game> getGame(String gameId);
```

Comment:
Why do you need a precise interface? Can't the user classes just `extends DataService<Game>` ?